src/styles/base.css

```diff
@@ -62,7 +62,27 @@ abbr[title] {
   *,
   *::before,
   *::after {
-    transition: none !important;
-    animation: none !important;
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    transition-duration: 0.01ms !important;
```

Comment:
This reset code has been going around. It avoids breaking some types of transition/animation code. Don't think it matters for us, but I thought we might as well be safe.